dotnet/samples/GettingStartedWithAgents/Step2_Plugins.cs

```diff
@@ -26,7 +27,7 @@ public async Task UseChatCompletionWithPluginAgentAsync()
                 Instructions = HostInstructions,
                 Name = HostName,
                 Kernel = this.CreateKernelWithChatCompletion(),
-                ExecutionSettings = new OpenAIPromptExecutionSettings() { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions },
+                ExecutionSettings = new AzureOpenAIPromptExecutionSettings() { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions },
```

Comment:
Az above, this change is optional

Reply:
Correct, I did this as a work-around until Sergey was able to address a serialization bug that was blocking execution.  I'll revert.